### PR TITLE
ci: fix CI issues and improve make script

### LINKED ISSUE #558: quickstart tests error
**What happened**:
Error occus when running the quickstart tests in #557 . 
But it didn't interrupt execution and finally report an success. See https://github.com/mosn/layotto/runs/6384572067?check_suite_focus=true
image

**What you expected to happen**:
1. The quickstart tests run successfully
2. If it fails , it should interrupt the execution and report an error

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### LINKED ISSUE #561: ci: let `make all` command test all the quickstart docs
**What would you like to be added**:
- Let `make all` command test all the quickstart docs.
- `make all` don't have to build layotto, since we already build it when running the quickstart tests.

**Why is this needed**:
Contributors can `make all` and find the CI issues locally before pushing to remote repo

### PR DESCRIPTION
**Pull request by seeflood**
**What this PR does**:
fix CI issues and improve make script.
More specifically, fix #561 and #558 

**Which issue(s) this PR fixes**:

Fixes #561
Fixes #558

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: Makefile
Comment (Xunzhuo) [inline Makefile:246]: Do we need to run quickstart tests locally?

Comment (seeflood) [reply Makefile:246]: Currently the quickstart test is essentially integration test. Running the integration test locally can help find CI issues as soon as possible.
Just personal opinion, waiting for others' feedback

Comment (Xunzhuo) [reply Makefile:246]: I understand but I remember that quick start checks will generate some files, we should handle these generated files gracefully

Comment (seeflood) [reply Makefile:246]: Yes, I added them into the gitignore file. Maybe we can delete them before and after running the quickstart tests?

Comment (Xunzhuo) [reply Makefile:246]: Yes, you can try to delete them after testing finished or add them in make clean.

**Review by Xunzhuo — DISMISSED** [review]
LGTM, no need to over-think this, as long as it is benefical to confirm any mistakes in codes. I am OK with this.

File: Makefile
Comment (seeflood) [reply Makefile]: yep, I think you are right. I added clean script. please review again

**Review by zhenjunMa — APPROVED** [review]

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**
